bundles/io/org.eclipse.smarthome.io.console/src/main/java/org/eclipse/smarthome/io/console/internal/extension/MetadataConsoleCommandExtension.java

```diff
@@ -116,11 +119,20 @@ private void addMetadata(Console console, String itemName, String namespace, Str
         if (config == null) {
             return null;
         }
+        String configStr = config;
+        if (configStr.startsWith("{")) {
```

Comment:
Should we combine that to checks into one using "and"?

```Java
if (configStr.startsWith("{") && configStr.endsWith("}")) {
    configStr = configStr.substring(1, configStr.length() - 1);
}
```

I assume valid inputs are only start AND end with curly bracket or not using any.

Reply:
Sure, makes sense.